packages/react-utils/src/components/BrokenPage/tests/index.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { BrokenPage, useHandleBrokenPage } from '..';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router';
+import { HOME_URL } from '../../../../../product-catalogue/src/constants';
```

Comment:
I know this is a test but I think that **if possible** we should make each package completely self-reliant.